src/Simulation.cpp

```diff
@@ -321,7 +325,12 @@ void Simulation::readXML(XMLfileUnits& xmlconfig) {
 			}
 			else if(parallelisationtype == "DomainDecomposition") {
 				delete _domainDecomposition;
+				#ifndef MAMICO_COUPLING
 				_domainDecomposition = new DomainDecomposition();
+				#else
+				_domainDecomposition = new DomainDecomposition(coupling::interface::LS1StaticCommData::getInstance().getLocalCommunicator(),
+															coupling::interface::LS1StaticCommData::getInstance().getDomainGridDecomp());
+				#endif
```

Comment:
```suggestion
				_domainDecomposition = new DomainDecomposition(
#ifdef MAMICO_COUPLING	coupling::interface::LS1StaticCommData::getInstance().getLocalCommunicator(),
															coupling::interface::LS1StaticCommData::getInstance().getDomainGridDecomp()
#endif
				);
```
Only `ifdef` what's really necessary to make it easier to spot the difference